src/msg.c

```diff
@@ -1400,7 +1400,7 @@ static void start_timeout(struct l_timeout *timeout, void *user_data)
 	l_timeout_modify(timeout, 5);
 }
 
-static void append_queue(void *data, void *user_data)
+static void append_update_queue(void *data, void *user_data)
```

Comment:
The name of this function is not clear enough. Please, update it.

Reply:
For me, it is clear enough. What do you suggest?

Reply:
i change to append_on_update_list